packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/SimpleDocument.tsx

```diff
@@ -139,7 +145,11 @@ export const SimpleDocument: FC<Props> = ({
         />
       </div>
     </div>
-  ) : null;
+  ) : (
+    <div className={base}>
+      <ErrorView />
+    </div>
+  );
```

Comment:
Don't duplicate the wrapper div:

```suggestion
  return (
      <div className={base}>
          {html ? {
              <div className={`${base}__wrapper`}> ...
          ) : (
              <ErrorView />
          )}
      </div>
  );
```